README.md

```diff
@@ -111,7 +111,7 @@ Installs the mixlib-install/mixlib-install gems and upgrades the chef-client.
 - `exec_args` - An array of arguments to exec the chef-client with. default: ARGV
 - `download_url_override` - The direct URL for the chef-client package.
 - `checksum` - The SHA-256 checksum of the chef-client package from the direct URL.
-- `upgrade_delay` - The delay in seconds before the scheduled task to upgrade chef-client runs on windows. default: 30
+- `upgrade_delay` - The delay in seconds before the scheduled task to upgrade chef-client runs on windows. default: 61. When using values 60 seconds or less, there is a risk to schedule upgrade within the same minute that Chef runs and upgrade won't be triggered.
```

Comment:
This information is applicable only to windows users and is a detailed edge case that most users will not need to care about.  It should probably be moved to a comment.

Reply:
A more simple "(lowering this limit is not recommended)" would be better here.